portability-types-transfer/src/main/java/org/datatransferproject/types/transfer/auth/AuthData.java

```diff
@@ -18,4 +18,6 @@
 import org.datatransferproject.types.common.PortableType;
 
 /** Base type for authentication data. */
-public abstract class AuthData extends PortableType {}
+public abstract class AuthData extends PortableType {
+  public abstract String getToken();
```

Comment:
Can we add a bit of documentation?  Also, do we want to return null or throw an exception for subclasses that do not support this?

Reply:
Throwing an exception may be the more sensible thing to do for subclasses that explicitly do not support this. That means it won't fail silently but it can be caught and handled if wanted.

Reply:
Pushed a commit to throw an UnsupportedOperationException